.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+# Based on GitHub Actions CMake template
+name: CMake
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
That's  a good start but before we can rely on it, we need more coverage including Windows and macOS builds.

Reply:
This is the list of available native runners: https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners - are we good with `windows-2019` + `ubuntu-20.04`  + `macos-10.15`?

Reply:
I'd go with the respective -latest aliases but otherwise, I think we should be fine.